website/docs/components/table/partials/code/component-api.md

```diff
@@ -1,49 +1,107 @@
-- The `Hds::Table::Td` component is a template-only component. It supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering) and can contain interactive elements (e.g., `<td><a href="user-info.html">User info</a></td>`) but is not eligible to receive interactions itself (e.g., it cannot have an `onClick` event handler attached directly to it).
+### Table
 
 <Doc::ComponentApi as |C|>
   <C.Property @name="<:head>" @type="named block">
-    This is a named block where the content for the table head (`<thead>`) is rendered.
+    This is a named block where the content for the table head (`<thead>`) is rendered. It is unlikely that most consumers will need to use this named block directly for most use cases.
   </C.Property>
   <C.Property @name="<:body>" @type="named block">
     This is a named block where the content for the table body (`<tbody>`) is rendered.
   </C.Property>
   <C.Property @name="model" @type="array">
-    If defined, sets the data source that gets yielded by the `:body` named block.
+    Indicates the data model to be used by the table.
   </C.Property>
   <C.Property @name="columns" @type="array">
-    Use an `array` hash to define your table columns. While `key` and `label` are required, other options include `isSortable`, `align` (for text-alignment), and `width`.
+  Use a `hash` within the array to define each column. `Key` and `label` values are both required. Add `isSortable` if you would like the column to be sortable.
   </C.Property>
   <C.Property @name="sortBy" @type="string">
-    If defined, indicates which column should be pre-sorted when the table is rendered.
+    If defined, the value should be set to the key of the column that should be pre-sorted when the table is rendered.
   </C.Property>
   <C.Property @name="sortOrder" @type="string" @values={{array "asc" "desc" }} @default="asc">
     Use in conjunction with `sortBy`. If defined, indicates which direction the column should be pre-sorted in. If not defined, `asc` is applied by default.
   </C.Property>
-  <C.Property @name="onSort" @type="function">
-    Callback function invoked (if provided) when one of the sortable table headers is clicked, and a sorting is triggered. The function receives the values of `sortBy` and `sortOrder` as arguments.
-  </C.Property>
   <C.Property @name="isStriped" @type="boolean" @values={{array "false" "true" }} @default="false">
     Define on the table invocation. If set to `true`, row striping ("zebra striping") will be applied to the table.
   </C.Property>
-  <C.Property @name="isFixed" @type="boolean" @values={{array "false" "true" }} @default="false">
+  <C.Property @name="isFixedLayout" @type="boolean" @values={{array "false" "true" }} @default="false">
     If set to `true`, the `table-display`(CSS) property will be set to `fixed`, which will automatically distribute columns equally based on the total width of the table.
```

Comment:
I just noticed this, while reading, and I don't think is correct: the columns are not always equally distributed, it depends if some of them have a width applied (in which case the width is attempted to be preserved)

https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout#values
`fixed` - Table and column widths are set by the widths of table and col elements or by the width of the first row of cells.

Reply:
It was the easiest/most terse way to explain it. But I can add info about the browser's algorithm if you really think it would be useful.